appcenter-data/Data.js

```diff
@@ -23,11 +23,13 @@ const Data = {
         if (readOptions === undefined) {
             readOptions = new Data.ReadOptions(TimeToLive.DEFAULT);
         }
-        return AppCenterReactNativeData.read(documentId, partition, readOptions).then((result) => {
-            // Create a new `Date` object from timestamp as milliseconds
-            result.lastUpdatedDate = new Date(result.lastUpdatedDate);
-            return result;
-        });
+        return AppCenterReactNativeData.read(documentId, partition, readOptions).then(convertTimestampToDate);
+    },
+    create(documentId, document, partition, writeOptions) {
```

Comment:
Can you test how we can write "Javadoc"? Would it be visible in VS code when mousing over?

Reply:
Added it looks something like this 
<img width="768" alt="Screen Shot 2019-06-26 at 2 58 44 PM" src="https://user-images.githubusercontent.com/8866892/60218087-1976bc80-9823-11e9-88cd-d3ef9c449327.png">